lib/configuration.yaml

```diff
@@ -259,6 +259,7 @@ mainnet_base: &mainnet_base
         avvmDistr: {}
 
     dbSerializeVersion: 0
+    requiresNetworkMagic: NMMustBeNothing
```

Comment:
Could this be put in the `mainnet_full` and `mainnet_dryrun_full` sections but not in `mainnet_base`?

That is, only set NMMustBeNothing in the "leaf" configuration sections.

Otherwise `testnet_full` and others will pick up that setting.

Or I can also change it in #3585.

Reply:
Sorry about the delayed reply on this. 

If we modify the configuration on #3659, we’ll have to wait for CI to pass again before merging. I can push a commit to #3585 to ammend the configuration, since that will need to be rebased once #3659 is merged anyways. Would that work?

Reply:
Change of plans, ignore the above! We're going to modify the configuration, since we have to cherry-pick over the configuration test from https://github.com/input-output-hk/cardano-sl/pull/3265/commits/2b0187f3228dc223f1c2c4eb9caa10fb46bdf495, which will require a re-run of CI anyways.

Reply:
This should be resolved as of https://github.com/input-output-hk/cardano-sl/pull/3659/commits/99d6bb1b58cd05c45857e34c5148cde6ee4ddee2